File: C Programming/Competitive Programming/Coding Challenges 6/Program1.c
#include<stdio.h>

void Number(int iNo)
{
    if (iNo<0)
    {
        printf("\n Invalid Number: %d \n",iNo);
       
    }
    else if (iNo<50)
    {
        printf("\nSmall Number: %d \n",iNo);
    }
    else if (iNo > 50 && iNo <100)
    {
        printf("\nMedium Number : %d \n",iNo);
    }
    else
    {
        printf("\nLarge Number: %d \n",iNo);
    }
}

int main()
{
    int iValue =0;

    printf("\nEnter number : ");
    scanf("%d",&iValue);

    Number(iValue);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 12/program1.java
import java.util.Scanner;

class program1
{
    static void Number(int iNo)
    {
        String sResult;

        if (iNo < 0)
        {
            sResult = "Invalid Number";
        }
        else if (iNo <= 50)
        {
            sResult = "Small Number";
        }
        else if (iNo < 100)
        {
            sResult = "Medium Number";
        }
        else
        {
            sResult = "Large Number";
        }

        System.out.println("\n" + sResult + ": " + iNo + " \n");
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("\nEnter number : ");
        iValue = sobj.nextInt();

        Number(iValue);

        sobj.close();
    }
}